octobot_backtesting/collectors/exchanges/exchange_collector.pxd

```diff
@@ -26,8 +26,19 @@ cdef class ExchangeDataCollector(data_collector.DataCollector):
     cdef public object tentacles_setup_config
     cdef public object start_timestamp
     cdef public object end_timestamp
+    cdef public bint finished
+    cdef public bint in_progress
+    cdef public int current_time_frame_index
+    cdef public int total_time_frame
```

Comment:
What's the `total_time_frame` ?

Reply:
number of time_frame muliplied by number of symbol

Reply:
I think we should name it differently because it is not very easy to understand. Do you have an idea? I do not have any :smile:

Reply:
`total_step` and `current_step` ?

Reply:
sounds good !